lib/tessel/deploy.js

```diff
@@ -205,49 +211,98 @@ actions.sendBundle = function(tessel, filepath, opts) {
 
 actions.tarBundle = function(opts) {
   var target = opts.target || process.cwd();
+  var pack = tar.Pack();
+  var buffers = [];
+
+  if (opts.slim) {
+    console.log('Running slim...');
```

Comment:
Sure thing. Those were mostly for development, but while refactoring, I'll use the `logs` utility instead.